### PR TITLE
[CI:DOCS]Add search endpoint

### PR DESCRIPTION
**Pull request by baude**
add search endpoint with golang bindings

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
could someone please merge this?

File: pkg/api/server/register_images.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/containers/libpod/pkg/api/handlers"
 	"github.com/containers/libpod/pkg/api/handlers/generic"
 	"github.com/containers/libpod/pkg/api/handlers/libpod"
+	"net/http"
```
Comment (rhatdan) [inline pkg/api/server/register_images.go]: Wrong spot and eliminate line afterwards.

**Comment by baude** [comment]
rebased and ready

**Pull request merged**